lib/shared/permissions.js

```diff
@@ -101,7 +116,22 @@ exports.getEnvironmentCommandPrefix = (environment) => {
     }
 
     if (isWindows) {
-      return [ 'set', `${key}=${value}`, '&&' ];
+      const definition = `${key}=${value}`;
+      return [
+        'set',
+        _.attempt(() => {
+          if (/\s/.test(definition)) {
+            return quoteString(definition);
+          }
+
+          return definition;
+        }),
+        '&&'
+      ];
+    }
+
+    if (/\s/.test(value)) {
```

Comment:
@jviotti what do you think about reusing the `command-join` module to escape these, instead of custom code?